.github/workflows/library_dafny_verification.yml

```
@@ -28,7 +28,7 @@ jobs:
             AwsCryptographicMaterialProviders,
             StandardLibrary,
           ]
-        os: [macos-latest-large]
+        os: [macos-12-large]
```

We should leave this one alone.
It does not run any code, just verifies it.
```suggestion
        os: [macos-latest-large]
```